src/middlewared/middlewared/api/v25_04_0/smartctl.py

```diff
@@ -10,6 +10,7 @@ class AtaSelfTest(BaseModel):
     status_verbose: str
     remaining: float
     lifetime: int
+    power_on_hours_ago: int # current lifetime - lifetime at test start, "power on hours ago"
```

Comment:
Now this comment isn't necessary :)
Also PEP8 should fail here, here must be two spaces between the code and the `#`